.gitignore

```diff
@@ -0,0 +1,2 @@
+test.md
+node_modules
```

Comment:
- [ ] [REQUIRED] This pull request can be improved by updating the title and description.
Pull Request should have a descriptive title telling what changes are inside.
Pull Request should have a summary describing introduced changes in more detail.
e.g Title -> Hello Microverse project and the description -> Mention the things you did in brief detail.